RIOT-OS/RIOT
# boards/nucleo-g431rb & g474re: add pinouts to boards doc page

### Contribution description

This PR adds pinouts to the documentation pages for:
-  `nucleo-g431rb`
-  `nucleo-g474re`

### Testing procedure

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-g431rb.html
xdg-open doc/doxygen/html/group__boards__nucleo-g474re.html
```

### Issues/PRs references

None
